src/Interactive/Host/Interactive/Core/InteractiveHost.Service.cs

```diff
@@ -367,44 +358,42 @@ private static string GenerateUniqueChannelLocalName()
             /// Reads given initialization file (.rsp) and loads and executes all assembly references and files, respectively specified in it.
             /// Execution is performed on the UI thread.
             /// </summary>
-            [OneWay]
-            public void InitializeContext(RemoteAsyncOperation<RemoteExecutionResult> operation, string initializationFile, bool isRestarting)
+            public void InitializeContext(string initializationFile, bool isRestarting)
```

Comment:
>void [](start = 19, length = 4)

This will need to be `async Task InitializeContext` (all remote APIs need to be async now).